rainforest/tests.go

```diff
@@ -197,6 +197,23 @@ func (c *Client) GetRFMLIDs() (TestIDMappings, error) {
 	return testResp, nil
 }
 
+// GetTest gets a test from RF specified by the given test ID
+func (c *Client) GetTest(testID int) (*RFTest, error) {
+	req, err := c.NewRequest("GET", "tests/"+strconv.Itoa(testID), nil)
```

Comment:
its probably better to just use fmt.Sprintf(...) in these cases @epaulet , but just a minor nitpick